aws/table_aws_shield_protection_group.go

```diff
@@ -0,0 +1,269 @@
+			},
+		},
+		Columns: awsGlobalRegionColumns([]*plugin.Column{
+			{
+				Name:        "protection_group_id",
+				Description: "The name of the protection group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "aggregation",
+				Description: "Defines how Shield combines resource data for the group in order to detect, mitigate, and report events.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "pattern",
+				Description: "The criteria to use to choose the protected resources for inclusion in the group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "resource_type",
+				Description: "The resource type to include in the protection group. All protected resources of this type are included in the protection group.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ResourceType").Transform(transform.NullIfZeroValue),
+			},
+			{
+				Name:        "arn",
+				Description: "The ARN (Amazon Resource Name) of the protection group.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ProtectionGroupArn"),
+			},
+			{
+				Name:        "members",
+				Description: "The ARNs (Amazon Resource Names) of the resources that are included in the protection group.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Members").Transform(transform.EnsureStringArray),
+			},
+			{
+				Name:        "tags_src",
+				Description: "The list of tags associated with the protection group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     listTagsForShieldProtectionGroup,
+				Transform:   transform.FromField("Tags"),
+			},
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ProtectionGroupId"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ProtectionGroupArn").Transform(arnToAkas),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     listTagsForShieldProtectionGroup,
+				Transform:   transform.From(handleShieldProtectionGroupTurbotTags),
+			},
+		}),
+	}
+}
+
+//// HYDRATE FUNCTIONS
+
+func listAwsShieldProtectionGroups(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := ShieldClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_shield_protection_group.listAwsShieldProtectionGroups", "connection_error", err)
+		return nil, err
+	}
+
+	if svc == nil {
+		// Unsupported region, return no data
+		return nil, nil
+	}
+
+	// Limiting the results
+	queryResultLimit := int32(1000)
```

Comment:
```suggestion
	queryResultLimit := int32(10000)
```
https://docs.aws.amazon.com/waf/latest/DDOSAPIReference/API_ListProtectionGroups.html